FileWithInputData.csv

```diff
@@ -0,0 +1 @@
+type;fruit ;quantity
b;banana;20
b;apple;100
b;orange;25
s;banana;100
p;banana;13
r;apple;10
p;apple;20
s;orange;50
p;banana;5
s;banana;50
p;orange;35
b;melon;0
s;melon;200
p;melon;150
r;melon;10
b;raspberry;345
b;pineapple;15
p;pineapple;4
```

Comment:
Make stack structure for input file

Reply:
![image](https://github.com/mate-academy/jv-fruit-shop/assets/156338484/373797f5-976e-4e05-9b9c-1da0a23cd09b)

Reply:
Use prepared content